src/utils.tsx

```diff
@@ -187,9 +188,9 @@ export const fetchService = async (url: string, requestMethod: HttpRequestMethod
   });
 
   const { status } = response;
-  if (status === 401 || status === 403) onFailCallback?.();
+  if (status === 401 || status === 403) throw new AuthError('');
```

Comment:
Would it be doable to add some kind of message in here for the AuthError? I figure maybe the error field of the response might return something useful?